<?xml version="1.0" encoding="utf-8"?>
<launch>
    <node name="rviz" pkg="rviz" type="rviz" output="log" args="-d $(find am_traj)/config/traj.rviz"/>

    <arg name="topic_poly_traj" value="/rnw/poly_traj" />

    <node pkg="rnw_ros" type="visualize_traj_node" name="traj_visualization" output="screen" required="true">
        <remap from="~poly_traj" to="$(arg topic_poly_traj)"/>
        <param name="lift_dt" type="double" value="0.05"/>
        <param name="length_g" type="double" value="0.5"/>
    </node>

</launch>
